app/views/storage_service/new.html.haml

```diff
@@ -0,0 +1,6 @@
+= render :partial => "layouts/flash_msg"
+
+- if @storage_manager
+  = react 'StorageServiceForm', :redirect => previous_breadcrumb_url, :storageManagerId => @storage_manager.id.to_s
+- else
+  = react 'StorageServiceForm', :redirect => previous_breadcrumb_url
```

Comment:
this works. but maybe we can 

- avoid the if statement here and handle the props in the component
- and by doing so, we can change the .haml extension to .erb

```
<%= render :partial => "layouts/flash_msg" %>
<%= react 'StorageServiceForm', :redirect => previous_breadcrumb_url, :storageManagerId => @storage_manager ? @storage_manager.id.to_s : null %>

```

Reply:
- u meant `nil` at the end there right?
- aren't the props already handled in component? this change seems to work without making adjustments to other files.